app/controllers/application_controller.rb

```diff
@@ -1,3 +1,5 @@
+require Rails.root.join('app', 'apis', 'ghn_client.rb')
```

Comment:
why you need to add this line here?

Reply:
on the second look, I now realize its unnecessary, it is then removed